src/containers/GraphSettingsContainer.tsx

```diff
@@ -87,7 +87,7 @@ class GraphSettings extends React.PureComponent<GraphSettingsProps> {
         // @ts-ignore
         GraphDataActions.fetchGraphData(
           store.getState().namespaces.activeNamespace,
-          this.props.graphDuration,
+          store.getState().userSettings.duration,
```

Comment:
Same note here: `GraphSettings` is already wrapped in a container. We should revert back to using `this.props` and use `mapStateToProps` to bind to the state to follow redux patterns.